Nasal/electrical.nas

```diff
@@ -79,7 +84,7 @@ BatteryClass.new = func {
 #
 
 BatteryClass.apply_load = func (amps, dt) {
-    var old_charge_percent = getprop("/systems/electrical/battery-charge-percent");
+    var old_charge_percent = me.charge_percent;
```

Comment:
Could you elaborate on this change? The reason it reads from  a property is so that there's no hidden state in Nasal (`me.charge_percent`) and the flight recorder can record the property. (Add a `debug.dump(old_charge_percent);` statement to test)